src/applications/edu-benefits/tests/5495/containers/IntroductionPage.unit.spec.jsx

```diff
@@ -1,30 +1,67 @@
 import React from 'react';
 import { expect } from 'chai';
 import { shallow } from 'enzyme';
+import { IntroductionPage } from 'applications/edu-benefits/5495/containers/IntroductionPage';
+import {
+  WIZARD_STATUS_NOT_STARTED,
+  WIZARD_STATUS_COMPLETE,
+  getWizardStatus,
+} from 'applications/static-pages/wizard';
+import { sessionStorageSetup } from 'platform/testing/utilities';
 
-import { IntroductionPage } from '../../../5495/containers/IntroductionPage';
+describe('the Edu-Benefit 5495 Introduction Page', () => {
+  let defaultProps;
 
-describe('Edu 5495 <IntroductionPage>', () => {
-  it('should render', () => {
-    const tree = shallow(
-      <IntroductionPage
-        route={{
-          formConfig: {},
-        }}
-        saveInProgress={{
-          user: {
-            login: {},
-            profile: {
-              services: [],
-            },
+  before(() => {
+    sessionStorageSetup();
+  });
+
+  beforeEach(() => {
+    defaultProps = {
+      shouldEduBenefits5495WizardShow: true,
+      route: {
+        formConfig: {},
+      },
+      saveInProgress: {
+        user: {
+          login: {},
+          profile: {
+            services: [],
           },
-        }}
-      />,
-    );
-    expect(tree.find('FormTitle').props().title).to.contain('Manage');
-    expect(tree.find('withRouter(Connect(SaveInProgressIntro))').exists()).to.be
-      .true;
-    expect(tree.find('.process-step').length).to.equal(4);
-    tree.unmount();
+        },
+      },
+    };
+  });
+
+  afterEach(() => {
+    sessionStorage.clear();
+  });
+
+  it('should show the wizard on initial render if shouldEduBenefits5495WizardShow is set to true', () => {
+    const wrapper = shallow(<IntroductionPage {...defaultProps} />);
+    expect(wrapper.exists('WizardContainer')).to.equal(true);
+    expect(wrapper.exists('.subway-map')).to.equal(false);
+    wrapper.unmount();
+  });
+  it('should show the subway map on initial render if shouldEduBenefits5495WizardShow is set to false', () => {
+    const props = {
+      ...defaultProps,
+      shouldEduBenefits5495WizardShow: false,
```

Comment:
This isn't how you should test a component that has mapStateToProps and is leveraging a selector - you should pass in state via a provider and ensure that the prop is set the way you want it there and that it makes it down to the component as desired - simply setting the prop doesn't test the mapping is working